node/src/node_configurator/configurator.rs

```diff
@@ -2551,7 +2551,7 @@ mod tests {
             .consuming_wallet_private_key_params(&consuming_wallet_private_key_params_arc)
             .consuming_wallet_private_key_result(Ok(Some(consuming_wallet_private_key.clone())))
             .mapping_protocol_result(Ok(Some(AutomapProtocol::Igdp)))
-            .max_block_count_result(Ok(Some(100000)))
+            .max_block_count_result(Ok(None))
```

Comment:
I think I lack the understanding of why this needs to be changed to `None` from a value of `100000`.